### DISCUSSION TITLE
[Projects Beta] Support logical OR query

### DISCUSSION THREAD

**Discussion by creasty** [community/community#6199] [Projects and Issues]
Relating to https://github.com/github/feedback/discussions/6198.
I'd like to create a view like `assigned:USERNAME OR review-requested:USERNAME`.

**Comment by tjdavis3** [comment]
Similarly, I'd like to be able to create a view as `repo:org/repo1 OR repo:org/repo2`

**Reply by OJFord** [reply]
Similarly multiple statuses (or other enum field) without listing all the excludes (which may be more than the included disjunction, and would also need to be edited if a new option was added).
